### DISCUSSION TITLE
Request or Response Specification Definition Document？

### DISCUSSION THREAD

**Discussion by StringKe** [trpc/trpc#3150] [Q&A] [answered]
I am developing a backend service using trpc, but I need to dispatch trpc in another programming language.

I would like to know the design specification of trpc related requests and responses so that I can implement it in other languages.

https://discord.com/channels/867764511159091230/1040096677727588362

**Comment by sachinraja** [comment, marked as answer]
See https://trpc.io/docs/rpc
